lib/InjectorServer.js

```diff
@@ -15,7 +15,9 @@ function injectorServer(options) {
     // Get previously cached mirror if existed
     mirror = resolveCachedMirror(cache, mirror);
 
-    var serializer = makeMirrorSerializer(true);
+    var serializer = makeMirrorSerializer(true, {
+      maxStringLength: 10000
```

Comment:
Perhaps we should use `-1` instead? AFAIK that way V8 does not truncate the response values at all.